frontend/.env.test

```
@@ -0,0 +1 @@
+REACT_APP_BASE_URL=http://my.test.serv
```

```suggestion
REACT_APP_BASE_URL=http://localhost:8080
```

Pretty sure you don't have a testserver :D

---

Yes, i'm also sure :)